### ISSUE TITLE
Json resume theme path must contain `jsonresume-theme-` somewhere

### ISSUE THREAD

**Issue by ael-code** [hacksalot/HackMyResume#173]
I'm trying to pass an absolute path as `--theme` argument.
First I discovered that you cannot use path that contains uppercase letters #172 and then I found that the path to a jsonresume theme must contain the string "jsonresume-theme-" somewhere.

In fact in the [`_loadTheme`](https://github.com/hacksalot/HackMyResume/blob/3cf850ea0e956ca5162c792ad8dc03cc621cf79a/src/verbs/build.coffee#L350) there is the following check.

```
  # Create a FRESH or JRS theme object
  theTheme =
    if _opts.theme.indexOf('jsonresume-theme-') > -1
then new JRSTheme().open(tFolder) else new FRESHTheme().open( tFolder );
```

This constraint is not specified anywhere in the docs and it seems to me a very bad way to recognize the type of the theme. In fact it make impossible to use a generic path for a jsonresume theme.

**Comment by hacksalot** [comment]
Yeah, that's ugly. Once upon a time we only supported certain predefined JRS themes IIRC. We'll roll this into 1.9.0.

**Comment by hacksalot** [comment]
Fixed in 231357badcc1fb4121b45b7f416b0386fd49f3b7.
